src/jwplayer/players/shared/JwPlayerWrapper.tsx

```diff
@@ -90,9 +90,11 @@ const JwPlayerWrapper: React.FC<JwPlayerWrapperProps> = ({
 
 			setConfig(config);
 
+			const { image, ...configWithoutImage } = config; // eslint-disable-line
```

Comment:
todo: let's add the comment regarding this line with an explanation why we do it this way

Reply:
Added a short description about it